docs/sources/mimir/release-notes/v2.7.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Grafana Mimir version 2.7 release notes"
+menuTitle: "V2.7 release notes"
+description: "Release notes for Grafana Mimir version 2.7"
+weight: 300
+---
+
+# Grafana Mimir version 2.7 release notes
+
+Grafana Labs is excited to announce version 2.7 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Store-gateway streaming enabled by default** The new default value of `5000` for `-blocks-storage.bucket-store.batch-series-size` enables store-gateway streaming in the default configuration. This means that series are loaded from object storage in batches rather than buffering them all in memory before returning to the querier. Enabling streaming can improve memory utilization in the store-gateway.
```

Comment:
[nit] I would clarify what "improve memory utilization" means:

```suggestion
- **Store-gateway streaming enabled by default** The new default value of `5000` for `-blocks-storage.bucket-store.batch-series-size` enables store-gateway streaming in the default configuration. This means that series are loaded from object storage in batches rather than buffering them all in memory before returning to the querier. Enabling streaming can reduce memory utilization peaks in the store-gateway.
```